packages/client/src/views/CorrectionForm/CorrectionSummary.tsx

```diff
@@ -145,9 +145,15 @@ class CorrectionSummaryComponent extends React.Component<IFullProps, IState> {
         this.props.offlineResources.config.CURRENCY &&
         this.props.offlineResources.config.CURRENCY['isoCode']
     })[0].symbol
-    ;(
-      this.group.fields[0].nestedFields as any
-    ).REQUIRED[0].label.defaultMessage += ` ${currency}`
+
+    if (
+      (this.group.fields[0].nestedFields as any).REQUIRED[0].label
+        .defaultMessage === 'Total'
```

Comment:
I think we need french translation instead of hard coded value

Reply:
Updated the code. Farajaland PR link: